projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/order-history/b2b-order-history-orders-flow.e2e-spec.ts

```diff
@@ -15,6 +15,7 @@ import {
   interceptOrdersEndpoint,
   waitForResponse,
 } from '../../../../helpers/order-history';
+import * as cart from '../../../../helpers/cart';
```

Comment:
Fyi, there were also implementation suggestions/requirements from QA in the ticket comments.  Please consider those as well.